examples/data-objects/table-tree/src/Table/DataObject.ts

```diff
@@ -0,0 +1,71 @@
+/*!
```

Comment:
Nit: most of the other files in this package are named using `camelCase`, but this and `Utils.ts` are named using `PascalCase`. Ideally, we should be consistent here. Git unfortunately won't understand if you rename the files _only_ with case changes, but you could probably do `tableDataObject.ts` and `utilities.ts` (or something) as a workaround.